### ISSUE TITLE
Unable to run Quemu Appliances on Gns3 server - Empty console

### ISSUE THREAD

**Issue by dani135** [GNS3/gns3-server#1443]
I have installed a remote GNS3 server over Ubuntu 16.04 following this guide https://docs.gns3.com/1c2Iyiczy6efnv-TS_4Hc7p11gn03-ytz9ukgwFfckDk/

All is working properly except Quemu appliances, no error when boot VM's but console don´t show output.

I have tried differents VM's with formats vmdk or qcow2, same result.

Svm flag on /proc/cpuinfo.

Checking quemu logs i just see **warning: host doesn't support requested feature: CPUID.01H:ECX.vmx [bit 5]**

can anybody help me?

**Comment by grossmj** [comment]
@dani135 what version of Qemu do you have?

**Comment by grossmj** [comment]
@dani135 I remember some of our users reporting issues with Qemu version 2.5. Is there any way you run Ubuntu 18.04 LTS which comes with Qemu version 2.11? This could solve the problem.

**Comment by grossmj** [comment]
@dani135 do you still have this problem?

**Comment by 27jleon** [comment]
Hi, I install a remote GNS3 server with ubuntu 20.04
The installation has been completed successfully.
 About GNS3:
 I choose the option: run application on remote server. 
 I upload cisco IOS 7200 through Dynamic preference. I uploaded it and it was successful.
 However, I need upload image of mikrotik RouterOS and I have problem with QUEMU.
 
Output error is as follows:
error while starting MikroTikCHR7.11.2-1: KVM acceleration cannot be used (/dev/kvm doesn't exist). It is possible to turn off KVM support in the gns3_server.conf by adding enable_kvm = false to the [Qemu] section.
KVM acceleration cannot be used (/dev/kvm doesn't exist). It is possible to turn off KVM support in the gns3_server.conf by adding enable_kvm = false to the [Qemu] section.


When I used a local host to run GNS3, I installed VM GNS3 and resolved the problem. However, I not sure how to install a GNS3-VM on a remote server and interconnect it with it. 
Note: The server is a Dell R630, It has a environment that I must not affect. I was allowed to use a physical server space to set up a lab.

I hope you help me
